src/gluonts/model/lstnet/_network.py

```diff
@@ -202,44 +211,45 @@ def hybrid_forward(
             Shape (batch_size, num_series, 1) if `horizon` was specified
             and of shape (batch_size, num_series, prediction_length)
             if `prediction_length` was provided
-            
         """
 
-        scaled_past_target, _ = self.scaler(
+        scaled_past_target, scale = self.scaler(
```

Comment:
This is another point of confusion at the time, since the original implementation [normalizes globally](https://github.com/laiguokun/LSTNet/blob/master/utils.py#L21), but gluonts normalization is done locally for each sliding window and it can change the results drastically and at least as an approximation is better than not normalizing I think.

Reply:
yes so lstnet divides each time series by its max in the training window and then calculates the loss on the rescaled predictions and Ys. Here its rescaled in the sliding window... so yes not sure... I say we leave it like this.